### PR TITLE
Release notes 2022-08-30

### PR DESCRIPTION
**Pull request by seymourisdead**

### CODE REVIEW

File: content/docs/release-notes.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+## 2022-08-30 Console release
+
+### Bug fixes
+
+- UI: fix cors error for Swagger UI API requests.
+
+### What's new
+
+- UI: added Feedback form.
```
Comment (stepashka) [inline content/docs/release-notes.md:13]:
```suggestion
- UI: add Feedback form.
```

File: content/docs/release-notes.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 title: Release Notes
 ---
 
+## 2022-08-30 Console release
+
+### Bug fixes
+
+- UI: fix cors error for Swagger UI API requests.
```
Comment (ololobus) [inline content/docs/release-notes.md:9]:
```suggestion
- UI: fix CORS error for API requests in Swagger UI, when using 'Try it out'.
```
?

**Review by ololobus — APPROVED** [review]
Otherwise, LGTM

File: content/docs/release-notes.md (hunk 2 above)
Comment (ololobus) [reply content/docs/release-notes.md:9]: Should we mention oauth work? @duskpoet

Comment (duskpoet) [reply content/docs/release-notes.md:9]: nope, nothing to show to public yet

**Review by stepashka — APPROVED** [review]

**Pull request merged**